addons/microsoft_outlook/models/microsoft_outlook_mixin.py

```diff
@@ -161,6 +162,7 @@ def _generate_outlook_oauth2_string(self, login):
             if not self.microsoft_outlook_refresh_token:
                 raise UserError(_('Please connect with your Outlook account before using it.'))
             (
+                self.microsoft_outlook_refresh_token,
```

Comment:
Hello @std-odoo 

Does that ring a bell ?

Reply:
ah indeed, just tested and the refresh token change as well between 2 calls (the old refresh token still work, but I guess for a limited period of time), so better to re-update it

@schout-it Thanks for for fix :)

Reply:
@schout-it Shouldn't you fix it in 14.0 ?

Reply:
I can do it for 14.0 also, or you could backport
code is the same for all the versions from what I've seen